docs/markdown/CUDA_Driver_API_functions_supported_by_HIP.md

```diff
@@ -1063,8 +1064,8 @@
 | `cuFuncSetSharedMemConfig`                                |                               |
 | `cuLaunchKernel`                                          | `hipModuleLaunchKernel`       |
 | `cuLaunchHostFunc`                                        |                               | 10.0             |
-| `cuLaunchCooperativeKernel`                               |                               | 9.0              |
-| `cuLaunchCooperativeKernelMultiDevice`                    |                               | 9.0              |
+| `cuLaunchCooperativeKernel`                                | `hipLaunchCooperativeKernel` | 9.0              |
+| `cuLaunchCooperativeKernelMultiDevice`                    | `hipLaunchCooperativeKernelMultiDevice` | 9.0 |
```

Comment:
`cuLaunchCooperativeKernel` is not equal to `cudaLaunchCooperativeKernel `due to different signatures; the same is for `cuLaunchCooperativeKernelMultiDevice`.

Reply:
modified the doc.